Dockerfile

```diff
@@ -1,13 +1,19 @@
 FROM golang:1.15-buster as builder
 
-RUN DEBIAN_FRONTEND=noninteractive apt-get update -y && apt-get install -y --no-install-recommends libelf-dev llvm clang
+RUN echo "deb http://apt.llvm.org/buster/ llvm-toolchain-buster-9 main" >> /etc/apt/sources.list && apt-key adv --keyserver hkps://keyserver.ubuntu.com --recv-keys 15CF4D18AF4F7421 && \
+    DEBIAN_FRONTEND=noninteractive apt-get update -y && apt-get install -y --no-install-recommends libelf-dev llvm-9-dev clang-9 && \ 
+    (for tool in "clang" "llc" "llvm-strip"; do path=$(which $tool-9) && ln -s $path ${path%-*}; done)
 
 WORKDIR /tracee
 
 COPY . /tracee
 
+
+FROM builder as build
```

Comment:
Why not changing line 1 instead (builder->build)?

Reply:
it's not that builder changed to build. builder remains. build is a new stage. the is to speed up the make DOCKER=1 targets, so they don't need to build the binary just tp prep the builder image (the make will rebuild anyway).
BTW I have a commit that changes the Dockerfile (currently the prod docker image is broken), I just want to fix the build in github first